hazelcast/src/test/java/com/hazelcast/concurrent/lock/LockBasicTest.java

```diff
@@ -322,7 +322,7 @@ public void testLockLeaseTime_whenNullTimeout() {
 
     @Test
     public void testLockLeaseTime_lockIsReleasedEventuallyWhenPartitionIsMigrating() {
-        final InternalPartitionService ps = getNode(instances[0]).nodeEngine.getPartitionService();
+        final InternalPartitionService ps = getNode(instances[instances.length-1]).nodeEngine.getPartitionService();
```

Comment:
`getNode` on a proxied `HazelcastInstance` started by `HazelcastStarter` will return `null`. Instead, use `getNode` on the current `HazelcastInstance` which is guaranteed to be the last instance started by the compatibility `HazelcastInstance` factory.